pkg/agent/controller/agent.go

```diff
@@ -194,7 +194,7 @@ func (a *Controller) serviceExportToRemoteServiceImport(obj runtime.Object, op s
 	obj, found, err := a.serviceSyncer.GetResource(svcExport.Name, svcExport.Namespace)
 	if err != nil {
 		// some other error. Log and requeue
-		a.updateExportedServiceStatus(svcExport.Name, svcExport.Namespace, lighthousev2a1.ServiceExportInitialized,
+		a.updateExportedServiceStatus(svcExport.Name, svcExport.Namespace, mcsv1a1.ServiceExportValid,
```

Comment:
It seems odd to set it as `ServiceExportValid`. But the only other value is `ServiceExportConflict` which, the way they've defined it, is less appropriate. It seems the API  is a  bit lacking but it is what is.

Reply:
I think it is combined with Condition to mean something like "ServiceExportValid: Unknown" and finally set it to ready when it is truly valid.

Reply:
ok - still seems a bit odd tho, especially if one just looks at `ServiceExportValid`.  The code docs aren't clear on the usage (`ConditionStatus` has no docs) and, in fact, looks incorrect in some places. Anyway that's an upstream issue - maybe I'lll raise an  issue there.

Reply:
It’s only a prima facie state: it indicates that the export is syntactically valid, and will therefore be processed. Anything that happens after that isn’t reflected in the status, but in the message. It does seem rather odd, but is apparently in keeping with K8s practice elsewhere. It would still be worth bringing it up upstream...

Reply:
yeah - all the fields together make up the full status. It's OK just a bit different than what I'm used to.  I mean if I saw `ServiceExportValid` in output I'd think all is good but it may  not be - you also have to look at `ConditionStatus`. I'd expect a `ServiceExportFailed` state.  But it seems their design reasoning is that the field represents a category or type of condition to assert and the assertion of that condition is indicated by `ConditionStatus`, eg

- if we fail to lookup the service then we assert **ServiceExportValid: Unknown** meaning we can't determine if it's valid or not
- if the service is not exportable  or we fail to actually export it the we assert **ServiceExportValid: False** with the `Reason` providing more detail.